src/sql/platform/accounts/common/accountActions.ts

```diff
@@ -150,3 +151,26 @@ export class RefreshAccountAction extends Action {
 		}
 	}
 }
+
+/**
+ * Action to sign out of GitHub Copilot account
+ */
+export class GitHubCopilotSignOutAction extends Action {
+	public static ID = 'account.github.copilot.sign.out';
+	public static LABEL = localize('githubCopilotSignOut', "Sign out of GitHub Copilot");
+	public account?: azdata.Account;
+
+	constructor(
+		private _account: azdata.Account,
+		@IAuthenticationService private readonly authenticationService: IAuthenticationService
+	) {
+		super(GitHubCopilotSignOutAction.ID, GitHubCopilotSignOutAction.LABEL, 'remove-account-action codicon remove');
+	}
+
+	public override async run(): Promise<void> {
+		const providerId = this._account.key.providerId;
+		const allSessions = await this.authenticationService.getSessions(providerId);
+		const sessionsForAccount = allSessions.filter(s => s.account.label === this._account.displayInfo.userId);
+		await this.authenticationService.removeAccountSessions(providerId, this._account.displayInfo.userId, sessionsForAccount);
```

Comment:
Successfully signed out popup doesn't go away and requires user interaction, please make sure it goes away in 5-10 seconds.